src/test/java/bio/terra/service/filedata/azure/util/BlobIOTestUtility.java

```diff
@@ -136,12 +167,14 @@ public int read() throws IOException {
   }
 
   public void deleteContainers() {
-    destinationBlobContainerClient.delete();
+    destinationBlobContainerClient.ifPresent(BlobContainerClient::delete);
     sourceBlobContainerClient.delete();
   }
 
   public BlobContainerClientFactory createDestinationClientFactory() {
     return new BlobContainerClientFactory(
-        destinationBlobContainerClient.getAccountName(), tokenCredential, destinationContainerName);
+        destinationBlobContainerClient.orElseThrow().getAccountName(),
```

Comment:
duplicates `getDestinationBlobContainerClient()`
```suggestion
        getDestinationBlobContainerClient().getAccountName(),
```

Reply:
fair enough